client/src/main/ipc/custom.ts

```diff
@@ -118,7 +118,7 @@ For more information, check out the Serenade API documentation: https://serenade
     return new Promise((resolve) => {
       this.resolveStart = resolve;
       this.stop();
-      const stream = fs.createWriteStream(path.join(this.settings.path(), "serenade.log"));
+      const stream = fs.createWriteStream(path.join(this.settings.path(), "custom-command-server.log"));
```

Comment:
I changed the name of this file to be something different from the default since this is logging for another process, and the messages would be mixed.